doc/authoring_command_modules/authoring_commands.md

```diff
@@ -133,6 +133,8 @@ command(self, name, method_name=None, command_type=None, **kwargs)
 
 Any kwargs that are not specified will be pulled from the `command_type` kwarg, if present.
 
+For `show` command, ensure `404(Not Found)` is always trapped by either using `exception_handler=empty_on_404`, or catching the exception in your custom code. For context, refer to [issue](https://github.com/Azure/azure-cli/issues/6324).
```

Comment:
I think it makes more sense to put this guidance here as standard behavior for SHOW commands:
https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#standard-command-types